weave/scorers/context_relevance_scorer.py

```
@@ -0,0 +1,185 @@
+        self._tokenizer = AutoTokenizer.from_pretrained(
+            self._local_model_path,
+            model_max_length=self.model_max_length,
+        )
+        print(f"Model and tokenizer loaded on {self.device}")
+
+    def _score_document(
+        self, query: str, document: str, threshold: float
+    ) -> tuple[list[dict[str, Any]], int, int]:
+        """Score a single document."""
+        import torch
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)